astropy/io/misc/yaml.py

```diff
@@ -223,6 +237,7 @@ def _represent_tuple(self, data):
 AstropyDumper.add_multi_representer(u.StructuredUnit, _unit_representer)
 AstropyDumper.add_representer(tuple, AstropyDumper._represent_tuple)
 AstropyDumper.add_representer(np.ndarray, _ndarray_representer)
+AstropyDumper.add_representer(np.void, _void_representer)
```

Comment:
Does this also mean that a single row of a structured array can be represented? If so worth noting that in the change log.

Reply:
Done